gazettemarch18.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
- move this to the `news` folder
- rename this file to `2018-03-27-the-qat-gazette-march-2018.md`

Reply:
how do i move to folders